lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -454,8 +454,8 @@ function dosomething_user_user_pass_submit($form, &$form_state) {
         'uid' => $account->uid,
         'first_name' => dosomething_user_get_field('field_first_name', $account),
         'reset_link' => user_pass_reset_url($account),
-        'user_language'     => $account->language,
-        'user_country'      => isset($user_country_code) ? $user_country_code : 'US'
+        'user_language'     => isset($account->language) ? $account->language : 'en',
```

Comment:
Should we be using $account->language here or possibly using either dosomething_global_get_language(), or dosomething_global_get_user_language()? Both functions have some additional rules about which language should be chosen.